Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.5.4"
 HTML_Entities = "7693890a-d069-55fe-a829-b4a6d304f0ee"
 StrTables = "9700d1a9-a7c8-5760-9816-a99fda30bb8f"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
+GoogleDrive = "91feb7a0-3508-11ea-1e8e-afea2c1c9a19"
```

Comment:
The files seem to be on github. Why is google drive required?

Reply:
I first kept all files in G-Drive and later shifted it to Github
sure, I will remove GoogleDrive Dependency.

Reply:
so keeping `fetch_method = google_download` usually handle all kinds of downloads